### ISSUE TITLE
Quick image upload text field doesn't clean text

### ISSUE THREAD

**Issue by Tigerlady** [blueriver/MuraCMS#1896]
In the quick image upload the fields Summary Caption, Title, Credits & alt text.  If you copy and paste from word into these fields it doesn't just paste the plain text it brings in all the ugly xml code from word.  As much at you try to tell a client not to use word they do.  These fields should be plain text then if they want to edit the summary do it afterwards. 

Thanks. 

![quickupload-1](https://cloud.githubusercontent.com/assets/1850313/7991219/32194dfa-0abc-11e5-9880-e13194cb72ad.jpg)
![quickupload-titlefieldresults](https://cloud.githubusercontent.com/assets/1850313/7991218/32193c48-0abc-11e5-86a3-ba4c5280e1c0.jpg)

**Comment by jsieber-aca** [comment]
Just jumping on this thread as it also has caused issues for me as well. I'm wondering if ckeditor should even be used for these fields? I'm thinking that title, credits and alt text should be text inputs and if we want html in the Summary/Caption it should probably provide a minimum set of tools.

**Comment by Tigerlady** [comment]
I agree these fields should be just plain text inputs.
